kettle-plugins/common/namedClusterBridge/src/main/java/org/pentaho/big/data/plugins/common/ui/named/cluster/bridge/NamedClusterBridgeImpl.java

```diff
@@ -189,6 +202,21 @@ public NamedCluster clone() {
   }
 
   @Override
+  public String processURLsubstitution( String incomingURL, IMetaStore metastore, VariableSpace variableSpace ) {
+    if ( isMapr() ) {
+      String url = namedClusterManager
+        .processURLsubstitution( getName(), incomingURL, HadoopSpoonPlugin.MAPRFS_SCHEME, metastore, variableSpace );
+      if ( url != null && !url.startsWith( HadoopSpoonPlugin.MAPRFS_SCHEME ) ) {
+        url = HadoopSpoonPlugin.MAPRFS_SCHEME + "://" + url;
+      }
+      return url;
+    } else {
+      return namedClusterManager
```

Comment:
The NamedClusterManager at this line is the legacy NamedClusterManager-- is this intentional?  I would have expected it to leverage the processURLsubstitution from the new Impl class above.